src/util/response.js

```diff
@@ -0,0 +1,26 @@
+export function responseMessage(type, success, data, token){
+    let message = "Invalid Data";
+    if (success == true){
+        switch(type){
```

Comment:
Since you are doing a switch statement you probably want your default message to be what you put for line 2 and just declare message as a variable.

Reply:
the default message for the switch statement can't be "Invalid Data" because the if statement checks if the request was a success. I can do a default for get, post, put or delete though

Reply:
We need to handle error cases when success is false, cause we want to return error codes.